engine/class_modules/apl/apl_death_knight.cpp

```diff
@@ -41,7 +41,7 @@ std::string food( const player_t* p )
 
   if ( p->true_level >= 61 )
   {
-    frost_food  = ( p->dual_wield() ) ? "thousandbone_tongueslicer" : "sizzling_seafood_medley";
+    frost_food  = ( p->dual_wield() ) ? "sizzling_seafood_medley" : "sizzling_seafood_medley";
```

Comment:
If both 2h and dw are using the same food, probably best to remove the inline if statement and just have it as 
```suggestion
    frost_food  = "sizzling_seafood_medley";
```

Reply:
That's true, to be honest I only left it in to thoughtlessly swap it back in the future but might as well just remove it as it's not complicated anyway.